dietpi/dietpi-software

```diff
@@ -3551,6 +3551,15 @@ _EOF_
 
 			G_EXEC chmod +x /opt/dietpi-dashboard/dietpi-dashboard
 
+			cat << _EOF_ > /opt/dietpi-dashboard/config.toml
+port = 8080
```

Comment:
This port is in use already by Airsonic. To avoid conflicts, we need to use a different one, the next free one is `8088`. Also, we should not overwrite the config file if it exists already, to preserve user adjustments:
```sh
[[ -f '/opt/dietpi-dashboard/config.toml' ]] || cat << _EOF_ > /opt/dietpi-dashboard/config.toml
```

Many thanks for adding this, awesome! 👍

Reply:
8088 seems good, I'll set it as the default as well when nothing is specifically set.